spot_driver_cpp/src/kinematic/kinematic_service.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#include <spot_driver_cpp/kinematic/kinematic_service.hpp>
+
+#include <spot_driver_cpp/conversions/kinematic_conversions.hpp>
+#include <spot_driver_cpp/interfaces/rclcpp_logger_interface.hpp>
+#include <spot_driver_cpp/kinematic/default_kinematic_service_helper.hpp>
+
+namespace spot_ros2::kinematic {
+
+auto kServiceName = "get_inverse_kinematic_solutions";
```

Comment:
Make this a `static constexpr` member of the `KinematicService` class instead.

Reply:
I added `constexpr` and moved it inside an anonymous namespace. If I make it static I have to define it in the header file.